awscli/examples/s3/sync.rst

```diff
@@ -81,4 +81,4 @@ Output::
 
 The following ``sync`` command syncs files between two buckets in different regions::
 
-    aws s3 sync s3://my-us-west-2-bucket s3://my-us-east-1-bucket --source-region us-west-2 --region us-east-1
\ No newline at end of file
+    aws s3 sync s3://my-us-west-2-bucket s3://my-us-east-1-bucket --source-region us-west-2 --region us-east-1
```

Comment:
No, if you can successfully amend it you push the update to this branch. I suggest you just amend the commit and force push (it's your own fork, so yolo).